addons/web/controllers/main.py

```diff
@@ -1931,6 +1935,15 @@ class CSVExport(ExportFormat, http.Controller):
     def index(self, data, token):
         return self.base(data, token)
 
+    @http.route('/web/async_export/csv', type='json', auth="user")
+    def index_async(self, data, token):
+        model_desc = request.env[json.loads(data)['model']]._description
+        job = request.env['ir.async'].call_notify(
+            target=request.env['ir.async']._export_csv,
+            args=(json.loads(data),),
+            description=_("%s CSV export", _(model_desc)))
```

Comment:
Sorry, you can't translate the model description like that :cry:  Rightfully spotted by the linter!
You have to retrieve that stuff from the corresponding `ir.model`:
```python
model = request.env['ir.model]._get(json.loads(data)['model'])
model_desc = model.name        # this field is translated
```